Cargo.toml

```diff
@@ -1,6 +1,8 @@
 [package]
 name = "vm-memory"
 version = "0.1.0"
+description = "Safe abstractions for accessing the VM physical memory"
+keywords = ["memory, "virtual machine"]
```

Comment:
```suggestion
keywords = ["memory", "virtual machine"]
```

Reply:
Auch, nice catch :))) I would've need a PR to fix a PR that fixes a broken manifest :)) sounds familiar?